packages/babel-polyfill/src/index.js

```diff
@@ -1,7 +1,5 @@
 import "./noConflict";
 
-import global from "core-js/library/fn/global";
-
 if (global._babelPolyfill && typeof console !== "undefined" && console.warn) {
```

Comment:
Isn't `global` only needed here?

Reply:
ah! I had the global object proposal in mind, but it's not even name that way anymore. Let's close this PR.